splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceSchemaWatcher.java

```diff
@@ -35,21 +35,23 @@
     private static CleanupMode mode = CleanupMode.UNDEF;
     private static Semaphore sync;
 
+    public String dbName;
     public String schemaName;
     protected String userName;
 
-    public SpliceSchemaWatcher(String schemaName) {
+    public SpliceSchemaWatcher(String dbName, String schemaName) {
```

Comment:
Can we keep the original constructor `SpliceSchemaWatcher(String schemaName` and let it call `SpliceSchemaWatcher(null, schemaName)`? Maybe we can avoid touching so many ITs.

Reply:
+1, I was going to propose the same thing

Reply:
Good idea, this PR is now much smaller!